frames/src/main/java/com/checkout/frames/cvvcomponent/CVVComponentFactory.kt

```diff
@@ -0,0 +1,25 @@
+package com.checkout.frames.cvvcomponent
+
+import android.content.Context
+import com.checkout.base.model.Environment
+import com.checkout.frames.cvvcomponent.api.CVVComponentApi
+import com.checkout.frames.cvvcomponent.api.CVVComponentApiClient
+
+public object CVVComponentFactory {
```

Comment:
If the factory returns `CVVComponentApi`, shall we rename this to `CVVComponentApiFactory`? so it's easier to know that this is a factory for `CVVComponentApi`